btrfs.go

```diff
@@ -328,11 +328,11 @@ func SubvolDelete(path string) error {
 		}
 
 		if err := isFileInfoSubvol(fi); err != nil {
-			return err
+			return nil
```

Comment:
`isFileInfoSubvol` is used in cases where it is an error to call the function on something that is not a subvol. This was just an oversight in usage. Might be better to change to `(bool, error)` or something like that.